charts/dec-dtr/values.yaml

```diff
@@ -0,0 +1,304 @@
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+registry:
+  keycloak:
+    type: NodePort
+    args: [ "kc.sh import --file /opt/keycloak/data/import/default-realm-import.json; kc.sh start-dev --hostname-strict=false --proxy=edge" ]
+
+  enablePostgres: false
+  registry:
+    replicaCount: 1
+    imagePullPolicy: Never
+    imagePullSecrets:
+      - name: machineuser-pull-secret-ro
+    containerPort: 4243
+    host: localhost
+    ## If 'authentication' is set to false, no OAuth authentication is enforced
+    authentication: true
+    idpIssuerUri: http://registry-keycloak/realms/default-realm
+    idpClientId: default-client
+    tenantId: demo-tenant
+    service:
+      port: 8080
+      type: NodePort
+    dataSource:
+      driverClassName: org.postgresql.Driver
+      ## The url, user, and password parameter will be ignored if 'enablePostgres' is set to true.
+      ## In that case the postgresql auth parameters are used.
+      url: jdbc:postgresql://postgresql:5432/registry
+      user: registryuser
+      password: ioueohg8o734dfsd
+    ingress:
+      enabled: true
+      tls: false
+      urlPrefix: /registry
+      className: nginx
+      annotations:
+        cert-manager.io/cluster-issuer: letsencrypt-prod
+        nginx.ingress.kubernetes.io/rewrite-target: /$2
+        nginx.ingress.kubernetes.io/use-regex: "true"
+        nginx.ingress.kubernetes.io/enable-cors: "true"
+        nginx.ingress.kubernetes.io/cors-allow-credentials: "true"
+        nginx.ingress.kubernetes.io/x-forwarded-prefix: /registry
+    resources:
+      limits:
+        memory: "1024Mi"
+      requests:
+        memory: "512Mi"
+
+postgresql:
+  fullnameOverride: postgresql
+  service:
+    ports:
+      postgresql: 5432
+  auth:
+    postgresPassword: root
+  primary:
+    initdb:
+      scripts: 
+        init.sql: |
+          CREATE USER edcprovider WITH ENCRYPTED PASSWORD 'oqerhf8734hewrin';
+          ALTER USER edcprovider CREATEDB;
+          CREATE DATABASE provider;
+          ALTER DATABASE provider OWNER TO edcprovider;
+
+          CREATE USER edcconsumer WITH ENCRYPTED PASSWORD 'oqerhf8734hewrin';
+          ALTER USER edcconsumer CREATEDB;
+          CREATE DATABASE consumer;
+          ALTER DATABASE consumer OWNER TO edcconsumer;
+          
+          CREATE USER registryuser WITH ENCRYPTED PASSWORD 'ioueohg8o734dfsd';
+          ALTER USER registryuser CREATEDB;
+          CREATE DATABASE registry;
+          ALTER DATABASE registry OWNER TO registryuser;
+
+vault:
+  enabled: true
+  fullnameOverride: edc-vault
+  server:
+    dev:
+      enabled: true
+      devRootToken: "root"
+    postStart:
+      - "sh"
+      - "-c"
+      - |
+        { 
+        
+        sleep 15
+
+        /bin/vault kv put secret/data-encryption-aes-keys content=H7j47H6vVQQOv/hbdAYz+w==
+
+        cat << EOF | /bin/vault kv put secret/daps-private-key content=-
+        -----BEGIN PRIVATE KEY-----
```

Comment:
Please provide another way of provisioning secrets. Hard coded private keys and certificates in `values.yaml` won't be accepted.

Reply:
These certificates are needed to start out of the box.
Do you have a example template for how we can provide these certificates ?
Another solution would be that we do not offer the credentials at all and describe in the readme what still needs to be adjusted in the values.yml.